src/perf/lib/PerfClient.cpp

```diff
@@ -229,7 +229,8 @@ PerfClient::Init(
                 nullptr,
                 PerfClientConnection::TcpConnectCallback,
                 PerfClientConnection::TcpReceiveCallback,
-                PerfClientConnection::TcpSendCompleteCallback));
+                PerfClientConnection::TcpSendCompleteCallback,
+                TCP_EXECUTION_PROFILE_MAX_THROUGHPUT)); // TODO: Client should always be max tput?
```

Comment:
Client should default to lowlat but be configurable by command line too

Reply:
ACK, updated.